<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>791</width>
      <height>581</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="pageMainMenu">
     <widget class="QPushButton" name="btnFire">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>70</y>
        <width>61</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>üî•</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnGrowth">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>110</y>
        <width>61</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>üçì</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnIntrusion">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>150</y>
        <width>61</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>üö™</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnSensor">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>190</y>
        <width>61</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>üìä</string>
      </property>
     </widget>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>37</y>
        <width>401</width>
        <height>381</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="videoLayout">
       <item>
        <widget class="QWidget" name="videoWidget" native="true"/>
       </item>
       <item>
        <widget class="QPushButton" name="startCameraButton">
         <property name="text">
          <string>Start Camera</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="pageGrowth">
     <widget class="QPushButton" name="btnBackFromGrowth">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>190</y>
        <width>95</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string> home</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="pageIntrusion">
     <widget class="QPushButton" name="btnBackFromIntrusion">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>190</y>
        <width>95</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string> home</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="pageSensor">
     <widget class="QPushButton" name="btnBackFromSensor">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>180</y>
        <width>95</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string> home</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="pageFire">
     <widget class="QPushButton" name="btnBackFromFire">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>190</y>
        <width>95</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string> home</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
